docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -43,164 +50,198 @@ To use IBFT 2.0 requires an IBFT 2.0 genesis file. The genesis file defines prop
+
     ```bash
     curl -X POST --data '{"jsonrpc":"2.0","method":"ibft_getValidatorsByBlockNumber","params":["latest"], "id":1}' <JSON-RPC-endpoint:port>
-    ```  
- 
-To discard your proposal after confirming the validator was added, call `ibft_discardValidatorVote` specifying the address of the proposed validator.
+    ```
+
+To discard your proposal after confirming the addition of a validator, call
+[`ibft_discardValidatorVote`](../../../Reference/API-Methods.md#ibft_discardValidatorVote),
+specifying the address of the proposed validator.
 
 !!! example "JSON-RPC ibft_discardValidatorVote Request Example"
+
     ```bash
     curl -X POST --data '{"jsonrpc":"2.0","method":"ibft_discardValidatorVote","params":["0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73"], "id":1}' <JSON-RPC-endpoint:port>
     ```
+
 #### Removing a validator
 
-The process for removing a validator is the same as adding a validator except you specify `false` as the second parameter of `ibft_proposeValidatorVote`.
+The process for removing a validator is the same as adding a validator except you specify `false`
+as the second parameter of
+[`ibft_proposeValidatorVote`](../../../Reference/API-Methods.md#ibft_proposeValidatorVote).
 
 #### Epoch transition
 
-At each epoch transition, all pending votes collected from received blocks are discarded. Existing proposals remain 
-in effect and validators re-add their vote the next time they create a block. 
+At each epoch transition, IBFT 2.0 discards all pending votes collected from received blocks.
+Existing proposals remain in effect and validators re-add their vote the next time they create a
+block.
 
-An epoch transition occurs every `epochLength` blocks where `epochlength` is defined in the IBFT genesis file.
+An epoch transition occurs every `epochLength` blocks. Define `epochlength` in the IBFT 2.0 genesis
+file.
 
 ### Adding and removing validators without voting
 
-If network conditions mean voting cannot be used to change validators you can bypass voting and specify new
-validators in the genesis file. For example, a majority of the current validators are no longer
-participating in the network so a vote to add or remove valiators will never be successful. 
-
-!!! caution
-    Do not specify a transition block in the past. Specifying a transition block in the past may result in 
-    unexpected behaviour.  
+Network conditions might not allow voting to change validators. For example, if a majority
+of the current validators are no longer participating in the network, so a vote to add or remove
+valiators will never be successful. You can bypass voting and specify new validators in the genesis
+file.
 
 To add or remove validators without voting:
 
 1. Stop all nodes in the network.
-1. In the genesis file, add the `transitions` configuration item where: 
+1. In the genesis file, add the `transitions` configuration item where:
 
     * `<BlockNumber>` is the upcoming block at which to change validators.
```

Comment:
> ./docs/HowTo/Configure/Consensus-Protocols/IBFT.md:239: MD007 Unordered list indentation

One indentation step should be only 2 spaces.

Reply:
If only two or three spaces on the unordered list items, they show up in the rendered HTML as ordered list items. Four spaces they show up correctly.